apps/vmq_server/src/vmq_reg.erl

```diff
@@ -141,6 +141,23 @@ subscribe_op(vmq_reg_redis_trie, {MP, ClientId} = SubscriberId, Topics) ->
             {ok, []} ->
                 []
         end,
+    lists:foreach(
+        fun
+            ({[<<"$share">>, _Group | Topic], QoS}) ->
+                Key = {MP, Topic},
+                Value = {ClientId, QoS},
+                T1 = vmq_util:ts(),
+                ets:insert(vmq_shared_subs_local, {{Key, Value}}),
+                vmq_metrics:pretimed_measurement(
+                    {?LOCAL_SHARED_SUBS, insert},
+                    vmq_util:ts() - T1
+                ),
+                vmq_metrics:incr_cache_insert(?LOCAL_SHARED_SUBS);
```

Comment:
If you see the pretimed measurement fn also tracks the count. So do we actually need a separate counter given we are tracking the latency? 

Not sure whether we should be tracking the latency of ets table operations. Let me know how will it benefit us here.

Reply:
Does it? I don't remember this from the last time I had a look at this part of the code, what is the usual metric name correlation between counter and histogram?

Reply:
Changed